### DISCUSSION TITLE
Timestamp of words

### DISCUSSION THREAD

**Discussion by Archelunch** [snakers4/silero-models#11] [Q&A] [answered]
Hi! Is it possible to extract timestamps of words from your model's outputs?

**Comment by snakers4** [comment]
Yes you can just interpolate linearly from the outputs as a first approximation

@adamnsandle can you please add this as a flag to a naive decoder from the utils?

**Comment by snakers4** [comment]
Also note that there is torch audio vad in the example utils as well

**Comment by Archelunch** [comment]
How can I make interpolation? The lengths of output and input are different.

**Comment by snakers4** [comment]
we will push this feature to the baseline decoder we provide

**Comment by Archelunch** [comment]
Cool!

**Comment by snakers4** [comment, marked as answer]
https://github.com/snakers4/silero-models/pull/15

feature added to the PyTorch example in the colab notebook
[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/snakers4/silero-models/blob/master/examples.ipynb)
